packages/react-dom/src/__tests__/ReactDOMSelect-test.js

```diff
@@ -808,4 +808,184 @@ describe('ReactDOMSelect', () => {
     expect(node.options[1].selected).toBe(false); // b
     expect(node.options[2].selected).toBe(false); // c
   });
+
+  describe('When given a Symbol value', () => {
+    it('treats initial Symbol value as an empty string', () => {
+      let node;
+
+      expect(
+        () =>
+          (node = ReactTestUtils.renderIntoDocument(
+            <select onChange={noop} value={Symbol('foobar')}>
+              <option value={Symbol('foobar')}>A Symbol!</option>
+              <option value="monkey">A monkey!</option>
+              <option value="giraffe">A giraffe!</option>
+            </select>,
+          )),
```

Comment:
Style nit: but I think the implicit return with an assignment distracts from the test. Could you remove them, like:

```javascript
expect(() => {
  node = ReactTestUtils.renderIntoDocument(
    <select onChange={noop} value={Symbol('foobar')}>
      <option value={Symbol('foobar')}>A Symbol!</option>
      <option value="monkey">A monkey!</option>
      <option value="giraffe">A giraffe!</option>
    </select>
  )
}).toWarnDev('Invalid value for prop `value`');